### ISSUE TITLE
ChatGPT bugs

### ISSUE THREAD

**Issue by infiniumparty** [LucasAschenbach/prompster#18]
Hey, many times the extension doesn't open the terminal after I type / in ChatGPT, unless I reload the page. Maybe it is related to update where ChatGPT UI uses / for it's own commands like browse a web or use DallE?

**Comment by paraskarmacharya** [comment]
Yes, I've had the same issue. And now it is not working at all with the "/" options provided by ChatGPT as o1 model and websearch.

**Comment by LucasAschenbach** [comment]
In v1.4.6 you can now specify a custom trigger key, e.g. "#" which should fully resolve this issue.
